### DISCUSSION TITLE
Thanks for extrakto! 🎉

### DISCUSSION THREAD

**Discussion by ilyagr** [laktak/extrakto#107] [General]
I stumbled upon `extracto`, then realized I know you from https://github.com/gokcehan/lf.

It's an excellent tool! I appreciate how feature-full it is and it really speeds up my workflow. Thanks a lot!

This is not really an issue, feel free to close.

**Comment by dufferzafar** [comment]
@laktak I think it might be a good idea to enabled Discussions for this repository as well 😃

**Reply by laktak** [reply]
Good idea :) I guess discussions came after I created the repo so I never thought about enabling them.

**Comment by laktak** [comment]
@ilyagr Great to hear! Also thank you for your work on lf!
